pages/linux/ldapdomaindump.md

```diff
@@ -0,0 +1,20 @@
+# ldapdomaindump
+
+> Dumps users/computers/groups and OS/membership information via LDAP to HTML/JSON/greppable output.
+> More information: <https://github.com/dirkjanm/ldapdomaindump>.
+
+- Dump all information using the given LDAP account:
+
+`ldapdomaindump --user {{domain\\administrator}} --password {{password|NTLM_hash}} {{hostname|ip}}`
```

Comment:
```suggestion
`ldapdomaindump --user {{domain}}\\{{administrator}} --password {{password|NTLM_hash}} {{hostname|ip}}`
```

I wonder that `--user` option accepts only administrators?

Reply:
You're completely right, thank you for the head up.